nav2_costmap_2d/src/footprint_subscriber.cpp

```diff
@@ -68,9 +68,19 @@ FootprintSubscriber::getFootprint(
     return false;
   }
 
+  stamp = footprint_stamp;
```

Comment:
This is the difference, the stamp is now filled with actual footprint stamp inside getFootprint().